tests/brevitas/test_onnx_export.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import tempfile
+import unittest
+from pathlib import Path
+from typing import Dict
+
+import onnx
+import torch
+from brevitas.export.onnx.standard.qcdq.manager import StdQCDQONNXManager
+from brevitas_examples.llm.llm_quant.export import brevitas_proxy_export_mode
+from parameterized import parameterized
+from testing_utils import SUPPORTED_MODELS_TINY, VALIDATE_EXPORT_ON_SHAPES, get_quantized_model
+
+from optimum.exporters import TasksManager
+from optimum.exporters.onnx import (
+    export_models,
+    validate_models_outputs,
+)
+from optimum.exporters.onnx.utils import _get_submodels_and_onnx_configs
+from optimum.utils import DEFAULT_DUMMY_SHAPES
+from optimum.utils.testing_utils import grid_parameters
+from transformers.modeling_utils import get_parameter_dtype
+
+
+def _get_models_to_test(export_models_dict: Dict, library_name: str = "transformers"):
+    models_to_test = []
+    for model_type, model_names_tasks in export_models_dict.items():
+        model_type = model_type.replace("_", "-")
+        task_config_mapping = TasksManager.get_supported_tasks_for_model_type(
+            model_type, "onnx", library_name=library_name
+        )
+
+        if isinstance(model_names_tasks, str):  # test export of all tasks on the same model
+            tasks = list(task_config_mapping.keys())
+            model_tasks = {model_names_tasks: tasks}
+        else:
+            model_tasks = model_names_tasks  # possibly, test different tasks on different models
+
+        for model_name, tasks in model_tasks.items():
+            for task in tasks:
+                onnx_config_constructor = TasksManager.get_exporter_config_constructor(
+                    model_type=model_type,
+                    exporter="onnx",
+                    task=task,
+                    model_name=model_name,
+                    library_name=library_name,
+                )
+
+                models_to_test.append((f"{model_type}_{task}", model_type, model_name, task, onnx_config_constructor))
+    return sorted(models_to_test)
+
+
+def export_and_validate(
```

Comment:
In this function why not use: `onnx_export_from_model`?

Reply:
I need to have access to `models_and_onnx_configs`